src/js/background/backgroundLogic.js

```diff
@@ -93,7 +93,8 @@ const backgroundLogic = {
     // We can't open these we just have to throw them away
     if (protocol === "about:"
         || protocol === "chrome:"
-        || protocol === "moz-extension:") {
+        || protocol === "moz-extension:"
+        || protocol === "file:") {
```

Comment:
Are there other "privileged tabs" than `about`, `chrome`, `moz-extension` and `file`? See https://github.com/mozilla/multi-account-containers/issues/1717 for context.

Furthermore, https://github.com/mdn/webextensions-examples/blob/2184fee75afcde39328384b36adf9e52c5917949/apply-css/background.js only allows the `http` and `https` protocols. Should we maybe do the same and just whitelist all good protocols? If yes, which?